mobile/lib/common/settings_details.dart

```diff
@@ -0,0 +1,183 @@
+                      Text(
+                        "Build Info".toUpperCase(),
+                        style:
+                            const TextStyle(color: Colors.grey, fontSize: 18),
+                      ),
+                      const SizedBox(
+                        height: 10,
+                      ),
+                      Container(
+                          decoration: BoxDecoration(
+                              color: Colors.white,
+                              borderRadius: BorderRadius.circular(15)),
+                          child: Column(
+                            children: [
+                              moreInfo(context,
+                                  title: "Number",
+                                  info: widget.number,
+                                  lessInfo: true),
+                              moreInfo(context,
+                                  title: "Version",
+                                  info: widget.version,
+                                  lessInfo: true),
+                              moreInfo(context,
+                                  title: "Commit Hash", info: widget.branch),
+                              moreInfo(context,
+                                  title: "Branch", info: "Main", lessInfo: true)
+                            ],
+                          ))
+                    ],
+                  ),
+                ],
+              ),
+            ),
+          )),
+    );
+  }
+}
+
+Widget moreInfo(BuildContext context,
+    {required String title, required String info, bool lessInfo = false}) {
+  return Container(
+    padding: const EdgeInsets.all(15),
+    child: Row(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(
+              title,
+              style: const TextStyle(
+                  fontSize: 17,
+                  fontWeight: FontWeight.w400,
+                  color: Colors.black),
+            ),
+            const SizedBox(
+              height: 7,
+            ),
+            !lessInfo
+                ? SizedBox(
+                    width: MediaQuery.of(context).size.width - 100,
+                    child: Text(
+                      info,
+                      style: TextStyle(
+                          fontSize: 18,
+                          fontWeight: FontWeight.w300,
+                          color: Colors.grey.shade700),
+                    ))
+                : const SizedBox()
+          ],
+        ),
+        !lessInfo
+            ? GestureDetector(
+                onTap: () async {
+                  await Clipboard.setData(ClipboardData(text: info));
+                },
+                child: Icon(
+                  Icons.copy,
+                  size: 17,
+                  color: Colors.green.shade300,
```

Comment:
Can you make this `tenTenOnePurple` please? Feel free to use a `shade...` to make it look good.